app/models/comment.php

```diff
@@ -0,0 +1,55 @@
+<?php
+class Comment extends AppModel
+{
+    public $validation = array(
+        'body'          => array(
+            'length'    => array('validate_between', 1, 200),
+        ),
+    );
+
+    public function __construct(array $data = array())
+    {
+        parent::__construct($data);
+
+        if(!isset($this->user_id)) {
+            return;
+        }
+
+        $user = User::getById($this->user_id);
+        $this->username = $user->username;
+    }
+
+    public static function countAll($thread_id)
+    {
+        $db = DB::conn();
+        return $db->value("SELECT COUNT(*) FROM comment WHERE thread_id=?", array($thread_id));
+    }
+
+    public static function getAll($thread_id, $offset, $limit)
+    {
+        $db = DB::conn();
+        $rows = $db->rows("SELECT * FROM comment WHERE thread_id=? LIMIT {$offset}, {$limit}", array($thread_id));
```

Comment:
It worked. I thought it was deprecated because I saw '@DEPRECATED' when I looked in the SimpleDBI.php so I decided not to use it at first.